_config.yml

```diff
@@ -21,6 +21,8 @@ sass:
 baseurl: http://codefortucson.github.io/codefortucson-site
 url: http://www.codefortucson.org
 
+img_dir: /assets/images
```

Comment:
I have been thinking of doing this elsewhere and I think it would be worth doing as a separate PR to capture all of the instances. Maybe I'll go ahead and do that...